doc.go

```diff
@@ -20,10 +20,8 @@ existing, and then dying. This state model is required as creating large
 amounts of data can potentially take a long time to materialize. You are only
 able to interact with volumes that are in the middle state.
 
-Volumes are given a TTL which means that BaggageClaim can reclaim the disk
-space used by the volumes over the life of the service. Parent-child
-relationships between volumes are understood and will prevent a parent being
-deleted while a child is still active.
+Parent-child relationships between volumes are understood and will prevent a
```

Comment:
I don't think this is actually true - TTLs were the only way this was supported. I guess we should just remove the paragraph?

Reply:
@vito sure, can do. We do manage volume relationships on the ATC though. Is that documented anywhere? Should it be?

Reply:
BaggageClaim doesn't know anything about the ATC, but if you want the docs to reflect that here we could say that it's the responsibility of the API consumer to delete child volumes before parent volumes.

Reply:
included.